src/codegen/src/main/scala/DocGen.scala

```diff
@@ -38,12 +38,14 @@ object DocGen {
 
   def genRstFiles(): Unit = {
     // Generate a modules.rst file that lists all the .py files to be included in API documentation
-    val moduleString = allFiles(toZipDir, _.getName.endsWith(".py"))
-      .map(f => s"   ${f.getName.dropRight(3)}\n").mkString("")
+    val moduleString = allFiles(toZipDir, _.getName.endsWith(".py")).filterNot(_.getName.startsWith("_"))
+        .filterNot(_.getName.equals("java_params_patch.py"))
```

Comment:
No, the class descriptions is a different file/PR, which will not go out without more consultation with you. This PR addresses cleaning up modules.rst and the individual ClassName.rst files for the python API docs to deal with the questions I sent to Danil, Mark, and Ratan over the weekend regarding _CNTKModel.py vs CNTKModel.py, and the other _Class/Class pairs, as well as the java_params_patch.py file.